meilisearch/_httprequests.py

```diff
@@ -45,7 +45,7 @@ def send_request(
                     request_path,
                     timeout=self.config.timeout,
                     headers=self.headers,
-                    data=json.dumps(body) if body else "null",
+                    data=json.dumps(body) if body != None else "null",
```

Comment:
I'm sorry I asked you to make a change that doesn't work... oopsy
```suggestion
                    data=json.dumps(body) if body else "" if body == "" else "null",
```